modules/healthconnectonfhir/src/main/java/edu/stanford/healthconnectonfhir/ObservationToRecordMapperImpl.kt

```diff
@@ -0,0 +1,83 @@
+package edu.stanford.healthconnectonfhir
+
+import androidx.health.connect.client.records.BloodPressureRecord
+import androidx.health.connect.client.records.HeartRateRecord
+import androidx.health.connect.client.records.Record
+import androidx.health.connect.client.records.WeightRecord
+import androidx.health.connect.client.units.Mass
+import androidx.health.connect.client.units.Pressure
+import org.hl7.fhir.r4.model.Observation
+import java.time.ZoneOffset
+import javax.inject.Inject
+
+class ObservationToRecordMapperImpl @Inject constructor() : ObservationToRecordMapper {
```

Comment:
We can we mark this and `RecordToObservationMapperImpl` to internal

Reply:
I think this is not possible cause in `HealthConnectOnFhirModule`  it would say that public function exposes its internal parameter type.